lm_eval/tasks/realtoxicityprompts/metric.py

```diff
@@ -26,6 +29,14 @@ def toxicity_perspective_api(references, predictions, **kwargs):
     url = f"https://commentanalyzer.googleapis.com/v1alpha1/comments:analyze?key={API_KEY}"
 
     scores = []
+
+    failed = defaultdict(list)
```

Comment:
```suggestion
```

Reply:
so the reason why I wanted to do this, was to group failures. but i guess if we are just logging them we don't need this anymore